integrators/simple_harmonic_motion.cpp

```diff
@@ -205,7 +205,8 @@ TEST_P(SimpleHarmonicMotionTest, Error) {
   parameters_.sampling_period = 1;
   integrator_->SolveTrivialKineticEnergyIncrement<Length>(
```

Comment:
I don't know, but this file is due for speedy deletion, I'm tired of the obsolete SPRK and SRKN classes.